### DISCUSSION TITLE
Is it possible to not show the tool bar at bottom in exported html?

### DISCUSSION THREAD

**Discussion by GoBigorGoHome** [marp-team/marp#444] [Q&A] [answered]
by tool bar I mean 
![image](https://github.com/marp-team/marp/assets/9896288/5b93f0d1-64f4-471b-8b7e-11e89f87227d)

I am using marp in vscode.

**Comment by yhatt** [comment, marked as answer]
Currently VS Code extension cannot hide the on-screen controller from the slide, but you can hide it by using [Marp CLI](https://github.com/marp-team/marp-cli).

```
marp --bespoke.osc=false your-markdown.md
```
